.github/workflows/ci-develop-linux.yml

```diff
@@ -8,26 +8,64 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
 
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
 
-      - name: Install dependent packages
-        run: |
-          sudo apt update
-          sudo apt install powershell gdb xvfb
+      - name: Setup Yarn environment
+        uses: DerYeger/yarn-setup-action@v1.0.1
+        with:
+          node-version: 16
+
+      - name: Build the extension
+        uses: GabrielBB/xvfb-action@v1.6
+        with:
+          run: yarn compile-production
+
+      - name: Lint
+        run: yarn lint
+
+      - name: Install CMake/Ninja
+        uses: lukka/get-cmake@v3.18.2
```

Comment:
It must be 3.18.3 so that the PATH can be updated.

Reply:
3.18.3 breaks windows tests

https://github.com/microsoft/vscode-cmake-tools/runs/7437471573?check_suite_focus=true

Reply:
We need to find a different version then.  3.18.2 can't be used with this action because it fails the GitHub security check.  Can you repro the failure locally on Windows with 3.18.3?

Reply:
for mac, we get the cmake server error on 3.18.3.
cmake 3.18.6 worked for mac in another PR test: https://github.com/microsoft/vscode-cmake-tools/runs/7442887319?check_suite_focus=true but it hits error here

Reply:
3.19.0 in another PR also passed for Mac: https://github.com/microsoft/vscode-cmake-tools/runs/7441678664?check_suite_focus=true